### DISCUSSION TITLE
How to stop React from escaping quotes in inline scripts in static.config?

### DISCUSSION THREAD

**Discussion by crregions** [react-static/react-static#1609] [Q&A]
I'm trying to add the Google Tag Manager in the `static.config.js` but it seems to keep escaping quotes.

How do I add this script or any scripts properly here?

![image](https://user-images.githubusercontent.com/71287019/117559617-abfd5f00-b03b-11eb-9b6b-762111463eeb.png)

**Comment by Pixelatex** [comment]
There are some GTM packages you can use that sort this problem out quite easily.

Otherwise, you can also try using `dangerouslySetInnerHTML`
